README.md

```diff
@@ -25,3 +25,46 @@ As an example this is how you would include the dependency in maven:
     <classifier>linux-x86_64</classifier>
 </dependency>
 ```
+
+## FAQ
+
+### I tried to use io_uring but got `java.lang.RuntimeException: failed to create io_uring ring fd Cannot allocate memory`
+
+When you tried to use io_uring but saw an exception that looked like this please try to encreate the [memlock limit](https://access.redhat.com/solutions/61334) for the user that tries to use io_uring:
+
+
+```
+Exception in thread "main" java.lang.UnsatisfiedLinkError: failed to load the required native library
```

Comment:
Kind of odd from a users perspective that this becomes an UnsatisfiedLinkError.

Reply:
Well it happens during loading of the native lib, and its consistent with how we signal these failures back in other native libs in netty.